zebra-chain/src/sapling/spend.rs

```diff
@@ -10,17 +10,19 @@ use crate::{
     },
 };
 
-use super::{commitment, note, tree};
+use super::{commitment, note, tree, AnchorVariant, PerSpendAnchor};
 
 /// A _Spend Description_, as described in [protocol specification §7.3][ps].
 ///
 /// [ps]: https://zips.z.cash/protocol/protocol.pdf#spendencoding
-#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
-pub struct Spend {
+///
+/// TODO: remove anchor default.
```

Comment:
You might find the `Spend<SharedAnchor>` change easier if you convert to `Spend<PerSpendAnchor>` when using `Spend`s outside of `ShieldedData`:
https://github.com/ZcashFoundation/zebra/pull/1946/files#r602200961